### PR TITLE
Monkey refactor to a species

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Todo
- [x] finish converting stok sprites
- [x] finish converting naera sprites
- [x] test more
- [x] probably tm

## Why It's Good For The Game

One day we may kill /carbon
have fun reviewing this @SplinterGP 
Note this doesnt kill attackpaw that should be in a followup pr

## Changelog
:cl:
refactor: Monkeys are now at last a human species rather than a seperate type
fix: various monkey inconsistencies
/:cl:

### CODE REVIEW

**Comment by SplinterGP** [comment]
> have fun reviewing this @SplinterGP

You fool, i'm the ultimate mob code reviewer

**Comment by BraveMole** [comment]
Were monkies able to wear armor / use guns before this? They will have all the inventory slots as well now, right?

Will that not break most of the sprites?

File: code/game/objects/machinery/kitchen/gibber.dm [hunk 1]
```diff
@@ -147,17 +147,17 @@
...
-	else if( istype(src.occupant, /mob/living/carbon/) || istype(src.occupant, /mob/living/simple_animal/ ) )
+	else if(istype(occupant, /mob/living/carbon) || istype(occupant, /mob/living/simple_animal))
 
 		var/mob/living/carbon/C = occupant
 		var/sourcename = src.occupant.name
 		var/sourcenutriment = C.nutrition / 15
 		var/sourcetotalreagents = 0
 
-		if( istype(src.occupant, /mob/living/carbon/monkey/) || istype(src.occupant, /mob/living/carbon/xenomorph) ) // why are you gibbing aliens? oh well
+		if(istype(occupant, /mob/living/carbon/human) || istype(occupant, /mob/living/carbon/xenomorph)) // why are you gibbing aliens? oh well
```
Comment (SplinterGP) [inline code/game/objects/machinery/kitchen/gibber.dm:157]:
```suggestion
		if(ismonkey(occupant) || istype(occupant, /mob/living/carbon/xenomorph)) // why are you gibbing aliens? oh well
```

File: code/game/objects/machinery/kitchen/processor.dm [hunk 2]
```diff
@@ -48,39 +48,10 @@
...
-
-				bucket_of_blood.reagents.reagent_list += B
-				bucket_of_blood.reagents.update_total()
-				bucket_of_blood.on_reagent_change()
-				//bucket_of_blood.reagents.handle_reactions() //blood doesn't react
-				..()
-
-			input = /mob/living/carbon/monkey
-			output = null
+	/* mobs */  //there were here once but fuk that noise
```
Comment (SplinterGP) [inline code/game/objects/machinery/kitchen/processor.dm:51]:
```suggestion
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]
seems all good to me

**Comment by SplinterGP** [comment]
all good, everything that came with it is a feature.

**Pull request merged**